--- ../config.def.h	2022-08-23 17:44:38.132349673 +0530
+++ ./config.def.h	2022-08-23 17:55:16.663633709 +0530
@@ -95,27 +95,30 @@ unsigned int tabspaces = 8;
 
 /* Terminal colors (16 first used in escape sequence) */
 static const char *colorname[] = {
-	/* 8 normal colors */
-	"black",
-	"red3",
-	"green3",
-	"yellow3",
-	"blue2",
-	"magenta3",
-	"cyan3",
-	"gray90",
-
-	/* 8 bright colors */
-	"gray50",
-	"red",
-	"green",
-	"yellow",
-	"#5c5cff",
-	"magenta",
-	"cyan",
-	"white",
-
-	[255] = 0,
+    /* 8 normal colors */
+    [0] = "#2e3436", /* black   */
+    [1] = "#cc0000", /* red     */
+    [2] = "#4e9a06", /* green   */
+    [3] = "#c4a000", /* yellow  */
+    [4] = "#3465a4", /* blue    */
+    [5] = "#75507b", /* magenta */
+    [6] = "#06989a", /* cyan    */
+    [7] = "#d3d7cf", /* white   */
+                                  
+    /* 8 bright colors */
+    [8]  = "#555753", /* black   */
+    [9]  = "#ef2929", /* red     */
+    [10] = "#8ae234", /* green   */
+    [11] = "#fce94f", /* yellow  */
+    [12] = "#729fcf", /* blue    */
+    [13] = "#ad7fa8", /* magenta */
+    [14] = "#34e2e2", /* cyan    */
+    [15] = "#eeeeec", /* white   */
+                                   
+    /* special colors */
+    [256] = "#000000", /* background */
+    [257] = "#eeeeec", /* foreground */
+    [258] = "#050505", /* background */
 
 	/* more colors can be added after 255 to use with DefaultXX */
 	"#cccccc",
@@ -127,14 +130,22 @@ static const char *colorname[] = {
 
 /*
  * Default colors (colorname index)
- * foreground, background, cursor, reverse cursor
+ * foreground, background, cursor
  */
-unsigned int defaultfg = 258;
-unsigned int defaultbg = 259;
-unsigned int defaultcs = 256;
+unsigned int defaultfg = 257;
+unsigned int defaultbg = 256;
+unsigned int defaultcs = 257;
 static unsigned int defaultrcs = 257;
 
 /*
+ * Colors used, when the specific fg == defaultfg. So in reverse mode this
+ * will reverse too. Another logic would only make the simple feature too
+ * complex.
+ */
+unsigned int defaultitalic = 7;
+unsigned int defaultunderline = 7;
+
+/*
  * Default shape of cursor
  * 2: Block ("█")
  * 4: Underline ("_")
